src/backend/distributed/connection/remote_commands.c

```diff
@@ -337,10 +337,12 @@ ExecuteOptionalRemoteCommand(MultiConnection *connection, const char *command,
 
 
 /*
- * SendRemoteCommand is a PQsendQuery wrapper that logs remote commands, and
+ * SendRemoteCommand is a PQsendQueryParams wrapper that logs remote commands, and
  * accepts a MultiConnection instead of a plain PGconn.  It makes sure it can
  * send commands asynchronously without blocking (at the potential expense of
- * an additional memory allocation).
+ * an additional memory allocation). Note that since the function utilizes
+ * PQsendQueryParams() as the asynchronous command processing API, the command string
+ * could only include a single command.
```

Comment:
s/could/can/. I'd also just shorten the whole sentence to 'The command string can only include a single command', the reasoning isn't really relevant. If you want to hint at it, I'd just refer to `PQsendQueryParams` rather than including 'as the asynchronous command processing API' which isn't helpful.